CONTRIBUTING.md

```diff
@@ -71,33 +71,33 @@ This document describes how to contribute to ReVanced Manager.
 
 Features can be requested by opening an issue using the
-[feature request issue template](https://github.com/ReVanced/revanced-manager/issues/new?assignees=&labels=feature-request&projects=&template=feature-issue.yml&title=feat%3A+%3Ctitle%3E).
+[Feature request issue template](https://github.com/ReVanced/revanced-manager/issues/new?assignees=&labels=Feature+request&projects=&template=feature_request.yml&title=feat%3A+).
 
-> [!NOTE]
-> We may reject your request at the discretion of ReVanced Manager's maintainers,
-> please provide good motivation for a request to be accepted.
+> **Note**
+> Requests can be accepted or rejected at the discretion of maintainers of ReVanced Manager.
+> Good motivation has to be provided for a request to be accepted.
 
 ## 🐞 Submitting a bug report
 
-If you encounter a bug while using the ReVanced Manager app, open an issue using the
-[bug report issue template](https://github.com/ReVanced/revanced-manager/issues/new?assignees=&labels=bug&projects=&template=bug-issue.yml&title=bug%3A+%3Ctitle%3E).
+If you encounter a bug while using ReVanced Manager, open an issue using the
+[Bug report issue template](https://github.com/ReVanced/revanced-manager/issues/new?assignees=&labels=Bug+report&projects=&template=bug_report.yml&title=bug%3A+).
 
 ## 📝 How to contribute
 
-> [!TIP]
-> We recommend that you discuss your changes with 
-> the maintainers of ReVanced Manager before contributing. 
-> This will help you determine whether your change is acceptable.
-
-1. Fork the repository and create a new branch based off the `dev` branch
-2. Commit your changes
-3. Open a pull request to the `dev` branch and reference issues that your pull request closes
-4. The maintainers of ReVanced Manager will review and provide suggestions. 
-   Once your pull request is approved and merged, it will be included in the next release of ReVanced Manager
+1. Before contributing, it is recommended to open an issue to discuss your change
+   with the maintainers of ReVanced Manager. This will help you determine whether your change is acceptable
+   and whether it is worth your time to implement it
+2. Development happens on the `dev` branch. Fork the repository and create your branch from `dev`
+3. Commit your changes
+4. Submit a pull request to the `dev` branch of the repository and reference issues
+   that your pull request closes in the description of your pull request
+5. Our team will review your pull request and provide feedback. Once your pull request is approved,
+   it will be merged into the `dev` branch and will be included in the next release of ReVanced Manager
 
 ## 🤚 I want to contribute but don't know how to code
 
 Even if you don't know how to code, you can still contribute by 
 translating ReVanced Manager on [Crowdin](https://translate.revanced.app/).
 
-❤️ Thank you for considering contributing to ReVanced Manager.
+❤️ Thank you for considering contributing to ReVanced Manager,  
+ReVanced
```

Comment:
Why is this necessary?

Reply:
This PR introduces consistency with other repos. Other repos have it too, so I added it here as well.